.gitignore

```diff
@@ -4,7 +4,7 @@
 *.orig
 *.DS_Store
 build
-.coverage
+.coverage*
```

Comment:
@nedbat I'll put together a fix for that and tag you in the PR. It was added in to get rid of all of the coverage files that were being written as a part of running tox commands, but I'll update to include coveragerc